### ISSUE TITLE
Hunter: Snake Trap snakes de-aggroing

### ISSUE THREAD

**Issue by Anzillu** [dalaranwow/dalaran-wow#5210]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I set down snake trap on a mob or mobs, and back up to be in range attacked mode, when the snakes would come out, they would just follow me and not aggro onto the mob/mobs I was attacking.

**Expected behaviour**: Tell us what should happen instead
The snakes should be defensive like a pet on defensive, they attack what I attack, or if something attacks me, the snakes attack it.
**Steps to reproduce the problem**:

1. Make snakes defensive
2. Make them attack what you target/attack
3. 

**Include proofs for this behaviour**
I dont have proof, but this has happened multiple times in multiple different locations since I have learned the snake trap ability.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Summon object: #186670
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Anzillu** [comment]
Steps to reproduce:
1. Set down snake trap on enemy/enemies
2. back up(3-4) steps to be in range
3. snake trap spawns snakes that are passive and wont attack

**Comment by TeridanDW** [comment]
Thank you for the report @Anzillu, but this issue was already reported in #4913
